### DISCUSSION TITLE
blog/web-scraping-in-r/

### DISCUSSION THREAD

**Discussion by giscus** [AntoineSoetewey/statsandr#103] [General]
# blog/web-scraping-in-r/

Learn how to do web scraping in R through a real life example.

https://statsandr.com/blog/web-scraping-in-r/

**Comment by technocrat** [comment]
Fab explainer. Please tell us this is excerpted from your forthcoming book with more. (Says the man who doesn’t have to go through the skull sweat himself.)

**Reply by AntoineSoetewey** [reply]
Thanks for your feedback @technocrat!

It is more likely that I publish a book about statistics in R than web scraping in R, but I'll keep you in touch in case it changes 📖

**Comment by gcarterIT** [comment]
Great tutorial: brief, easy to follow, great intro to web scraping.

**Reply by AntoineSoetewey** [reply]
Thanks, glad you find it useful @gcarterIT!

**Comment by aarontimo** [comment]
This is brilliant! Thank you so much

**Reply by AntoineSoetewey** [reply]
You're welcome @aarontimo!
